src/components/inspectors/DurationExpression.vue

```diff
@@ -15,7 +15,7 @@
           <option v-for="period in periods" :key="period.name" :value="period">{{ $t(period.name) }}</option>
         </b-form-select>
       </b-input-group-append>
-
+      <small class="form-text text-muted">Select the duration of the timer</small>
```

Comment:
Since this component is only used in one place right now I felt that hard coding this text wasn't a big issue.

Reply:
Not sure if translations are part of the scope of this PR but should it be wrapped in $t?